### PR TITLE
Previous version validation string causes JSON validation error (due …

### PR DESCRIPTION
**Pull request by michaelcfanning**
…to /d escaping). Added correct stacks representation.

### CODE REVIEW

**Comment by michaelcfanning** [comment]
@lgolding @rtaket

File: src/Sarif.Sdk.sln [hunk 1]
```diff
@@ -122,11 +122,13 @@ Global
 		{66915F7B-9872-4390-AFED-DB8DE7B761BE}.Release|x86.ActiveCfg = Release|x86
 		{66915F7B-9872-4390-AFED-DB8DE7B761BE}.Release|x86.Build.0 = Release|x86
 		{F6727C72-C40C-4A6E-BAE0-23B4B5FE7DB1}.Debug|Any CPU.ActiveCfg = Debug|Win32
+		{F6727C72-C40C-4A6E-BAE0-23B4B5FE7DB1}.Debug|Any CPU.Build.0 = Debug|Win32
```
Comment (michaelcfanning) [inline src/Sarif.Sdk.sln]: This change ensures that the native project builds as part of Debug|Release configurations.

File: src/Sarif/GrammarFiles/Sarif.g4 [hunk 2]
```diff
@@ -104,7 +104,7 @@ toolInfo :
         three values for product version in the VS_VERSION_INFO structure. If the tool is a .NET
         application, this value SHOULD be the first three dotted version values of AssemblyVersion.
         }
-		@pattern {^\d+\.\d+\.\d+\.(-([A-Za-z0-9\-\.]+))?(\+[A-Za-z0-9\-\.]+)?$}
+		@pattern {[0-9]+\\.[0-9]+\\.[0-9]+(-[0-9A-Za-z-]+)?(\\+[0-9A-Za-z-]+)?}
```
Comment (michaelcfanning) [inline src/Sarif/GrammarFiles/Sarif.g4]: I believe our spec says that version SHOUDL be a semantic ver but does not need to be. Should we remove validation entirely? I guess this is a place where a warning in a SARIF validation service would be helpful. In fact, this is a good definition of what belongs there: analysis where the SHOULD standards are getting compromised, surfaced as warnings.

Comment (lgolding) [inline src/Sarif/GrammarFiles/Sarif.g4]: Yes, IMO the validation should be removed.

**Comment by lgolding** [comment]
:+1:

**Pull request merged**